src/lib/secureStorage.ts

```
@@ -0,0 +1,259 @@
+        data
+      )
+      
+      const decoder = new TextDecoder()
+      const jsonString = decoder.decode(decryptedBuffer)
+      return JSON.parse(jsonString)
+    } catch (error) {
+      console.error('Decryption error:', error)
+      return null
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)